### PR TITLE
Validate contact form

### PR DESCRIPTION
**Pull request by BB-Simon**
1. Add validation to the email input
2. Add on submit function to the form

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -208,3 +208,30 @@ cardBtn.forEach((btn) => {
...
+  const email = e.target.email.value;
+  const error = document.querySelector('.error');
+  const isSubmitOk = validateEmail(email);
+  if (isSubmitOk) {
+    error.style.display = 'none';
+  } else {
+    error.style.display = 'block';
+    error.innerHTML = 'Email must content only lower case charecters';
+  }
+});
```
Comment (AmaduKamara) [inline js/index.js]: - [ ] The form email input validation works fine but when the lowercase email is entered and all input fields have values, the form is supposed to be submitted but it's not submitting at the moment. Kindly work on it 👍🏽

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️ ♻️ ♻️

Hi @BB-Simon,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used git-flow correctly 👍🏽
✅  Linter checks passed 👍🏽

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly work on making the form data submitted. At the moment the validation is working but when valid data has been entered the form should proceed to submission. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: js/index.js (hunk 1 above)
Comment (BB-Simon) [reply js/index.js]: Hi @AmaduKamara ,
Thanks for you nice review, you mean i need to submit the form using fetch API to formspree?

**Comment by AmaduKamara** [comment]
Yeah, exactly.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @BB-Simon, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestion.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

* N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**